Basic FE project template/Gruntfile.js

```diff
@@ -2,7 +2,11 @@ module.exports = function (grunt) {
   grunt.initConfig({
     concat: {
       js: {
-        src: ["ui-src/js/test.js", "ui-src/js/utilities.js"],
+        src: [
+          "ui-src/js/test.js",
+          "ui-src/js/utilities.js",
+          "ui-src/js/increase-number.js",
```

Comment:
The name "increase-number.js" is too general, change it to something like "animated-counter.js"